consumer/ue_context_management.go

```diff
@@ -32,14 +33,23 @@ func UeCmRegistration(ue *amf_context.AmfUe, accessType models.AccessType, initi
 			InitialRegistrationInd: initialRegistrationInd,
 			Guami:                  &amfSelf.ServedGuamiList[0],
 			RatType:                ue.RatType,
-			// TODO: not support Homogenous Support of IMS Voice over PS Sessions this stage
-			ImsVoPs: models.ImsVoPs_HOMOGENEOUS_NON_SUPPORT,
+			ImsVoPs:                models.ImsVoPs_HOMOGENEOUS_NON_SUPPORT,
 		}
-		ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
+		gppAccessCtx, cancel := context.WithTimeout(ctx, 30*time.Second)
 		defer cancel()
 
-		_, httpResp, localErr := client.AMFRegistrationFor3GPPAccessApi.Registration(ctx,
-			ue.Supi, registrationData)
+		gppAccessCtx, span := tracer.Start(gppAccessCtx, "HTTP PUT udm/{ueId}/registrations/amf-3gpp-access")
```

Comment:
Question: In all of the previous changes (other files), you "reuse" the `ctx` variable (e.g., `ctx, span := tracer.Start(gppAccessCtx, "HTTP PUT udm/{ueId}/registrations/amf-3gpp-access")` but here you are using a different name (i.e., `gppAccessCtx`). Any specific reason for this? Why not just naming it `ctx` as in the other files?

Reply:
see comment above